cmd/query/app/static_handler.go

```diff
@@ -39,6 +40,7 @@ var (
 
 	// The following patterns are searched and replaced in the index.html as a way of customizing the UI.
 	configPattern   = regexp.MustCompile("JAEGER_CONFIG *= *DEFAULT_CONFIG;")
+	configJsPattern = regexp.MustCompile(`(?im)(^\s+)\/\/.*JAEGER_CONFIG_JS.*\n.*`)
```

Comment:
```suggestion
	configJsPattern = regexp.MustCompile(`(?im)(^\s+)\/\/\s*JAEGER_CONFIG_JS.*\n.*`)
```

Reply:
what is the purpose of trailing `.*`? Also, with the `m` mode, does `.*\n` guarantee single-line match?

Reply:
`.*` - my bad, you are correct `\s+` is best fits.
`m` is for multiline search. I assume that we must replace 2 lines: line with `JAEGER_CONFIG_JS` and following comment on the next line
```
// JAEGER_CONFIG_JS
// the line above may be replaced by user-provided JS file that should define a UIConfig function.
```